packages/ui-map-components/src/components/Table/getMapTableData.tsx

```diff
@@ -26,9 +26,9 @@ const processColumns = (serieses: Series[]) => {
       Header: 'Name',
       accessor: 'name',
       // eslint-disable-next-line react/prop-types
-      Cell: ({ value }: { value: string | number | boolean | undefined }) => (
-        <FirstColumnCell>{String(value)}</FirstColumnCell>
-      ),
+      Cell: (value: { value: string | number | boolean | undefined }) => {
+        <FirstColumnCell>{String(value)}</FirstColumnCell>;
+      },
```

Comment:
This one needs to be reverted back to regular parentheses, otherwise nothing gets returned here